src/System.Buffers.Primitives/System/Buffers/Position.cs

```diff
@@ -1,46 +1,44 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.Buffers;
 using System.Diagnostics;
-using System.Net.Http.Headers;
+using System.IO.Pipelines;
 using System.Runtime.CompilerServices;
-using System.Text;
 
-namespace System.IO.Pipelines
+namespace System.Buffers
```

Comment:
I think it should not be in System.Buffers namespace, but it's ok for now. We can decide on the namespace separately.